FIRST_TIME_SETUP.md

```diff
@@ -0,0 +1,70 @@
+# First Time Setup Guide
+
+This instructions install Python and Node on your machine. If you've worked with Python or Javascript projects on your machine, skip this instructions and proceed with the README.
+
+## Set up Python
+
+The DeepCell Label backend runs on Python. You can check if Python is installed by running:
+
+```bash
+python --version
+```
+
+If the command fails, install Python with Anaconda by following the installation instructions here: https://docs.anaconda.com/anaconda/install/index.html
+
+### Create a Python virtual environment
+
+We recommend installing Python dependencies in a virtual environment isolate the project from the rest of your machine. If you're using Anaconda, create a virtual environment named `deepcell-label` with this command: `conda create -n deepcell-label python=3.7`.
+
+Then activate the virtual environment with: `conda activate deepcell-label`.
+
+Now when you run `pip install -r requirements.txt`, all the dependencies will be installed in the virtual environment.
+
+You can deactivate the virtual environment with: `source deactivate`.
+
+Read more about virtual environments with conda here: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html.
+
+If you are not using Anaconda, set up a virtual environment with Python's built-in virtual environment manager `venv`: https://docs.python.org/3/tutorial/venv.html.
+
+## Setup Node.js (Javascript)
+
+As DeepCell Label is a web application, the frontend is written in JavaScript. We'll install Node.js, a server-side Javascript platform, and Yarn, a Javascript package manager.
+
+<!-- adapted from https://gist.github.com/Igormandello/57d57ee9a9f32a5414009cbe191db432 -->
+
+### Node
+
+- #### Node installation on Windows and macOS
```

Comment:
I found this a little disorienting because the instructions for the backend seem to be targeted at Mac/Ubuntu users. If we include Windows instructions, it might be best to test on that OS or have a disclaimer stating that it was designed for Mac/Ubuntu but should run fine multiple OS.